lib/windows/launcher/actions/appsActions.js

```diff
@@ -234,7 +268,12 @@ export function loadOfficialApps() {
     return dispatch => {
         dispatch(loadOfficialAppsAction());
         mainApps.getOfficialApps()
-            .then(apps => dispatch(loadOfficialAppsSuccess(apps)))
+            .then(apps => {
+                dispatch(loadOfficialAppsSuccess(apps));
+                apps.forEach(app => dispatch(
```

Comment:
I suppose eslint should guide you here: `apps.forEach(({ source, name, url }) => ...`